cypress/integration/WebInterface/Test Cases/Validations/PassingCodeCoverageTestCaseListPage.spec.ts

```diff
@@ -0,0 +1,96 @@
+import {OktaLogin} from "../../../../Shared/OktaLogin"
+import {CreateMeasurePage} from "../../../../Shared/CreateMeasurePage"
+import {MeasuresPage} from "../../../../Shared/MeasuresPage"
+import {EditMeasurePage} from "../../../../Shared/EditMeasurePage"
+import {MeasureGroupPage} from "../../../../Shared/MeasureGroupPage"
+import {Utilities} from "../../../../Shared/Utilities"
+import {MeasureCQL} from "../../../../Shared/MeasureCQL"
+import {TestCasesPage} from "../../../../Shared/TestCasesPage"
+import {TestCaseJson} from "../../../../Shared/TestCaseJson"
+
+
+let measureName = 'TestMeasure' + Date.now()
+let CqlLibraryName = 'TestLibrary' + Date.now()
+let randValue = (Math.floor((Math.random() * 1000) + 1))
+let testCaseTitle = 'Title for Auto Test'
+let testCaseDescription = 'DENOMFail' + Date.now()
+let testCaseSeries = 'SBTestSeries'
+let validTestCaseJson = TestCaseJson.TestCaseJson_Valid_w_All_Encounter
+let newMeasureName = measureName + randValue
+let newCqlLibraryName = CqlLibraryName + randValue
+let measureCQL = MeasureCQL.CQL_Multiple_Populations
+
+describe('Test Case Validations', () => {
+
+    beforeEach('Create Measure', () => {
+        CreateMeasurePage.CreateAPIQICoreMeasureWithCQL(newMeasureName, newCqlLibraryName, measureCQL)
+        OktaLogin.Login()
+        MeasuresPage.clickEditforCreatedMeasure()
+        cy.get(EditMeasurePage.cqlEditorTab).should('exist')
+        cy.get(EditMeasurePage.cqlEditorTab).should('be.visible')
+        cy.get(EditMeasurePage.cqlEditorTab).click()
+        cy.get(EditMeasurePage.cqlEditorTextBox).should('exist')
+        cy.get(EditMeasurePage.cqlEditorTextBox).should('be.visible')
+        cy.get(EditMeasurePage.cqlEditorTextBox).type('{enter}')
+        cy.get(EditMeasurePage.cqlEditorSaveButton).should('exist')
+        cy.get(EditMeasurePage.cqlEditorSaveButton).should('be.visible')
+        cy.get(EditMeasurePage.cqlEditorSaveButton).should('be.enabled')
+        cy.get(EditMeasurePage.cqlEditorSaveButton).click()
+        cy.wait(13500)
+        OktaLogin.Logout()
+        MeasureGroupPage.CreateProportionMeasureGroupAPI(false, false, 'Initial Population', 'Initial Population', 'Initial Population', 'Boolean')
+        OktaLogin.Login()
+    })
+    afterEach('Logout', () => {
+        OktaLogin.Logout()
+        Utilities.deleteMeasure(measureName, CqlLibraryName)
+
+    })
+
+    it('Validate Passing and Code Coverage tabs contain the initial "-" value, and displayes a percentage when Test Case is ran', () => {
```

Comment:
are we validating a specific expected percentage?

Reply:
We aren't waiting for specific percentages, yet, that part of the percentages stuff is supposed to come in a later story, is what Prateek was telling me. At this time, we just needed to validate that, initially, it shows that dash ("-") value and then some randomly generated value after the "Execute Test Case" button is clicked. I manually validated the changing values of what appears after the button is clicked and I just made the automation verify that the dash ("-") no longer is present after the "Execute Test Case" button is clicked.